### ISSUE TITLE
The install target ignores PREFIX

### ISSUE THREAD

**Issue by yurivict** [dail8859/NotepadNext#123]
My PREFIX is /usr/local, but it still installs the files into /usr:
```
/usr/bin/NotepadNext
/usr/share/applications/NotepadNext.desktop
/usr/share/icons/hicolor/scalable/mimetypes/NotepadNext.svg
```

**Comment by dail8859** [comment]
I'm not familiar myself with utilizing PREFIX. 

According to https://stackoverflow.com/a/7106823 and the comments, it looks like INSTALL_ROOT might be the built-in way of defining where it gets installed rather than PREFIX. I know the automated workflow uses `make install INSTALL_ROOT=AppDir` so I'm not sure the difference between PREFIX and INSTALL_ROOT.

**Comment by yurivict** [comment]
No, ```INSTALL_ROOT``` in qmake is stage directory, not prefix.

In ```src/NotepadNext/NotepadNext.pro``` you have ```/usr/``` hardcoded.
It should be like the above link suggests:
```
isEmpty(PREFIX) {
 PREFIX = /usr/local
}
TARGET = myapp
TARGET.path = $$PREFIX/
```

**Comment by dail8859** [comment]
@yurivict 

I'm not overly familiar with compiling on Linux.

How does one normally specify PREFIX when building other applications? Is this passed into `make`?

**Comment by dail8859** [comment]
@zocker-160 

Would using PREFIX fix the need for modifying the `.pro` files for the flathub build here?

**Comment by yurivict** [comment]
> How does one normally specify PREFIX when building other applications? Is this passed into make?

PREFIX=/usr/local is added to qmake's arguments.

**Comment by dail8859** [comment]
@yurivict

I don't have a Linux development machine available right now so taking what you've described I attempted to fix this in #124. See if that PR helps.

**Comment by yurivict** [comment]
With this patch files are installed into correct locations.

**Comment by zocker-160** [comment]
> Would using PREFIX fix the need for modifying the .pro files for the flathub build here?

@dail8859 yes, if using `PREFIX` works, I can remove that

**Comment by dail8859** [comment]
@zocker-160 Thanks for confirming. #124 has been merged into master now.
